internal/kuberesource/mutators.go

```diff
@@ -175,6 +176,23 @@ func AddPortForwarders(resources []any) []any {
 	return out
 }
 
+// AddPortForwardersForAllPorts adds a port-forwarder for each Service resource that forwardsall ports of that service.
+// This should never be used together with AddPortForwarders.
+func AddPortForwardersForAllPorts(resources []any, ports []int32) []any {
```

Comment:
Imho this should be the default behaviour of `AddPortForwarders`, no need to introduce a new function.